admin/lib/validator/restart-validator.js

```diff
@@ -11,7 +11,10 @@ const restartValidator = {}
 restartValidator.validateReason = (reason, specifyReason) => {
   const validationError = new ValidationError()
   if (reason === 'DNC' && specifyReason.length === 0) {
-    validationError.addError('didNotCompleteInfo', restartErrorMessages.didNotCompleteInfo)
+    validationError.addError('didNotCompleteInfo', restartErrorMessages.errorMessage)
+  }
+  if (reason === 'CLD' && specifyReason.length === 0) {
```

Comment:
@GeorgeChatzigiannis Would there be value in moving these arbitrary codes (CLD, DNC, etc) to a config file or somewhere more centralised?

Reply:
Removing magic strings is always a good idea!

Reply:
@pris54 @jon-shipley updated. please review once again